### PR TITLE
Change backtick to single quote

### PR DESCRIPTION
**Pull request by bdunne**

### CODE REVIEW

File: db/migrate/20180712122000_remove_host_provisioning.rb [hunk 1]
```diff
@@ -2,13 +2,13 @@ class RemoveHostProvisioning < ActiveRecord::Migration[5.0]
   class MiqApproval < ActiveRecord::Base
     self.inheritance_column = :_type_disabled
 
-    belongs_to :miq_request, :class_name => `RemoveHostProvisioning::MiqRequest`
+    belongs_to :miq_request, :class_name => 'RemoveHostProvisioning::MiqRequest'
```
Comment (jrafanie) [inline db/migrate/20180712122000_remove_host_provisioning.rb]: 😮

File: db/migrate/20180712122000_remove_host_provisioning.rb [hunk 2]
```diff
@@ -2,13 +2,13 @@ class RemoveHostProvisioning < ActiveRecord::Migration[5.0]
...
   end
 
   class MiqRequest < ActiveRecord::Base
     self.inheritance_column = :_type_disabled
 
-    has_many :miq_approvals, :dependent => :destroy, :class_name => `RemoveHostProvisioning::MiqApproval`
+    has_many :miq_approvals, :dependent => :destroy, :class_name => 'RemoveHostProvisioning::MiqApproval'
```
Comment (jrafanie) [inline db/migrate/20180712122000_remove_host_provisioning.rb]: http://gifs.chrisarcand.com/deploy-fix-to-production-tomhanks.gif

File: db/migrate/20180712122000_remove_host_provisioning.rb (hunk 1 above)
Comment (Fryguy) [reply db/migrate/20180712122000_remove_host_provisioning.rb]: WAT

**Comment by Fryguy** [comment]
cc @gmcculloug  WAT

**Comment by gmcculloug** [comment]
😭

**Pull request merged**